core/src/main/resources/defaults/blueprints/cdp-sdx-medium-ha-722.bp

```diff
@@ -35,11 +33,13 @@
           "ranger-RANGER_TAGSYNC-BASE",
           "zookeeper-SERVER-BASE",
           "knox-KNOX_GATEWAY-BASE",
```

Comment:
We wanted to add `ranger-RANGER_ADMIN-BASE` in the gateway, no?

Reply:
Guillermo should have done that already?

Reply:
https://github.com/hortonworks/cloudbreak/pull/8786

Reply:
ah ok.